downstream/modules/aap-hardening/con-aap-as-SOC.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// downstream/assemblies/assembly-aap-security-use-cases.adoc
+
+[id="con-aap-as-SOC_{context}"]
+
+= {PlatformName} as part of a Security Operations Center 
+
+[role="_abstract"]
+
+Protecting your organization is a critical task.
+Automating functions of your security operations center (SOC) can help you streamline security operations, response, and remediation activities at scale to reduce the risk and cost of breaches.
+{PlatformName} can connect your security teams, tools, and processes for more successful automation adoption and use.
+Learn how automation can help you safeguard your business and respond to growing security threats faster.
+
+link:https://www.redhat.com/en/resources/security-automation-ebook[Simplify your security operations center] provides an overview of the benefits to automating SOC operations, including use cases such as:
```

Comment:
Would it be useful to tell the reader that this is a Red Hat e-book?

Reply:
That sounds good to me! What do you think of something like: "For more information on the benefits of automating SOC operations, including use cases, see the Red Hat E-book link:https://www.redhat.com/en/resources/security-automation-ebook[_Simplify your security operations center_]. @ariordan-redhat